sku.0/sys.server/compiled/game/script/library/gcw.java

```diff
@@ -2691,6 +2691,10 @@ public static boolean gcwIsInvasionCityOn(String city) throws InterruptedExcepti
             CustomerServiceLog("gcw_city_invasion", "gcw.gcwIsInvasionCityOn: GCW City: " + city + " is not configured to run a city invasion. Function returning False.");
             return false;
         }
+        if (city == "dearic" && Boolean.parseBoolean(getConfigSetting("GameServer", "lifeday")) == true) {

```

Comment:
Should probably use `city.equals("dearic")` to compare the two strings.

Reply:
Also you should probably use the `checkConfigFlag()` method from the utils class instead.